js/renovation/button.tsx

```diff
@@ -82,8 +88,9 @@ export const viewFunction = (viewModel: Button) => (
             </div>
         ) || (
             <div className="dx-button-content">
-                {viewModel.icon}
+                {viewModel.iconPosition === 'left' && viewModel.icon}
                 {viewModel.text && <span className="dx-button-text">{viewModel.text}</span>}
+                {viewModel.iconPosition !== 'left' && viewModel.icon}
```

Comment:
We can extract the next condition into a variable `viewModel.iconPosition === 'left'`